templates/client-daemonset.yaml

```diff
@@ -191,7 +191,7 @@ spec:
                 -node-meta=pod-name:${HOSTNAME} \
                 -node-meta=host-ip:${HOST_IP} \
                 -hcl='leave_on_terminate = true' \
-                -disable-host-node-id=false \
+                -disable-host-node-id={{ .Values.client.disableHostNodeID | default false }} \
```

Comment:
Can you add bats tests in test/unit/client-daemonset.bats?

For the cases:
1) It defaults to false
2) It can be set to true

Reply:
@ndhanushkodi Just added bats tests for this. Let me know if this is ok.